contracts/system-contracts/hedera-token-service/KeyHelper.sol

```diff
@@ -6,6 +6,7 @@ import "./HederaTokenService.sol";
 
 abstract contract KeyHelper {
     using Bits for uint256;
+    // slither-disable-next-line uninitialized-state
```

Comment:
should we include a short description on why these are non-issues?

Reply:
Added.